features/proxy_config.feature

```diff
@@ -61,13 +55,17 @@ Feature: Proxy configuration
          \*   sudo ua config set apt_http_proxy=<value>
          \*   sudo ua config unset apt_http_proxy
          \*/
-        Acquire::http::Proxy ".*";
-        Acquire::https::Proxy ".*";
+        Acquire::http::Proxy::esm.ubuntu.com ".*";
+        Acquire::https::Proxy::esm.ubuntu.com ".*";
         """
         When I run `apt update` with sudo
         And I run `cat /var/log/squid/access.log` `with sudo` on the `proxy` machine
         Then stdout matches regexp:
         """
+        CONNECT esm.ubuntu.com:443
+        """
+        Then stdout does not match regexp:
+        """
         .*GET.*ubuntu.com/ubuntu/dists.*
         """
```

Comment:
This is the part that tests the new changes are actually working. I'd prefer if we had something a bit more specific here: Both `archive.ubuntu.com` and `security.ubuntu.com` I think should be contacted by apt during a `ua attach`. Can we make sure those urls specifically are not present?

Reply:
Yeah, updated.